src/ophys_etl/modules/decrosstalk/ophys_plane.py

```diff
@@ -263,6 +263,39 @@ def intersection_over_union(roi0: OphysROI,
     return float(ii)/float(uu)
 
 
+def compare_rois(roi0: OphysROI,
```

Comment:
It might be cleaner to implement `__eq__` on `OphysROI`. Then you can do 
```
assert roi1 == roi2
```
for example

Reply:
I thought about that. I specifically wanted something that allowed `roi0.valid_roi != roi1.valid_roi` (so that I can test that the only change that was made is that an ROI was flagged as `invalid` due to quality control). If feel like `__eq__` should test whether the ROIs are exactly equal.

Thoughts?

Reply:
I still feel like `__eq__` could be used, with a note added that it ignores `valid_roi`.

Reply:
I feel rather strongly that `__eq__` should test that the two are identical. Given that I'm only using this method for tests, how about I move it into the testing module and we can have this debate again if it becomes relevant in the actual codebase?